src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.time.LocalDate;
```

Comment:
Consider spacing out your imports, as per [guidelines](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements) 

```suggestion
import java.util.Scanner;
import java.util.ArrayList;

import java.io.File;
import java.io.FileWriter;
import java.io.IOException;
import java.io.FileNotFoundException;

import java.time.LocalDate;
```